### PR TITLE
Effect styling

### LINKED ISSUE #23: Add Volume Control
As a user, I want to be able to control the output volume of the instrument

### PR DESCRIPTION
**Pull request by mistercanderson**
# Description of Changes

- Style effect components
- Add volume + distortion knobs
- Refine the effects chain for better sound quality
- Link dolphin animation to effect knobs
- Style page to match wireframe design inspo

# Linked Issues

- closes #23

### CODE REVIEW

**Comment by tysnj** [comment]
This is a very lovely application now. I love the styling and find no issue with merging this branch after UAT.

**Review by tysnj — APPROVED** [review]
Solid work cleaning up the code while making the app look stellar!

File: src/components/App/App.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { useState, useEffect } from 'react';
 import * as Tone from 'tone';
-import { gsap } from "gsap";
```
Comment (Patfindley) [inline src/components/App/App.js:3]: nice catch

File: src/components/Key/Key.css [hunk 2]
```diff
@@ -59,7 +59,7 @@
 
 @media screen and (max-width: 1024px) {
   .sharp {
-    flex-grow: .3;
```
Comment (Patfindley) [inline src/components/Key/Key.css:62]: good idea for responsiveness!

File: src/components/Scene/Scene.js [hunk 3]
```diff
@@ -1,64 +1,75 @@
...
-    //   )
-    // } else {
-    //   return (
+  //   )
+  // } else {
+  //   return (
 
-    //   )
-    // }
-  }
```
Comment (Patfindley) [inline src/components/Scene/Scene.js:64]: yeah we dont need these

File: src/index.css [hunk 4]
```diff
@@ -16,10 +16,11 @@ html, body, #root {
 
 body {
   position: fixed;
-  background: black;
+  background: #5472EB;
```
Comment (Patfindley) [inline src/index.css]: def into this color change

**Review by Patfindley — APPROVED** [review]

**Pull request merged**